pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -38,7 +38,9 @@ const (
 // +kubebuilder:printcolumn:name="Paused",type="boolean",JSONPath=".status.paused",description="Whether the resource reconciliation is paused or not",priority=1
 // +kubebuilder:subresource:status
 
-// Alertmanager describes an Alertmanager cluster.
+// The `Alertmanager` custom resource definition (CRD) defines a desired [Alertmanager](https://prometheus.io/docs/alerting) setup to run in a Kubernetes cluster. It allows to specify many options such as the number of replicas, persistent storage and many more.
+//
+// For each `Alertmanager` resource, the Operator deploys a `StatefulSet` in the same namespace. When there are two or more configured replicas, the Operator runs the Alertmanager instances in high-availability mode.
```

Comment:
To be consistent with Prometheus, we could talk about AlertmanagerConfig selection (a follow-up PR is fine).

Reply:
Maybe something like

> The resource defines via label and namespace selectors which `AlertmanagerConfig` objects should be associated to the deployed Alertmanger instances.

Reply:
yes